CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this app will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [2.3.0]
+### Added
+- Support for PHP 7.1 and 7.2 - [#216](https://github.com/owncloud/files_texteditor/pull/216)
+### Changed
+- Set max version to 10.1 because core platform switches to semver - [#226](https://github.com/owncloud/files_texteditor/pull/226)
```

Comment:
This looks odd, because in this very PR we are setting max version to 10.
I guess there will be a follow-up PR to add this changelog PR to the changelog?
Or will that break the internet, the universe and everything?

Reply:
nice find. I've adjusted the message now without linking to a ticket